sf/cosmos/type/v1/type.proto

```diff
@@ -40,7 +40,7 @@ message TransactionData {
 }
 
 message MessageData {
-  Any message = 1;
+  google.protobuf.Any message = 1;
```

Comment:
Any reason we need the original `Any` type here? `MessageData` is created on the Graph Node side and we can pass our custom `Any` type there.

Reply:
It's because of the generated AscMessageData object from the type.proto. To keep everything in sync.

Reply:
Also don't have to create new object from 
```
message TxBody {
  repeated google.protobuf.Any messages = 1;
...
```